runatlantis.io/docs/server-configuration.md

```diff
@@ -718,6 +718,13 @@ Values are chosen in this order:
   environment where releases.hashicorp.com is not available. Directory structure of the custom
   endpoint should match that of releases.hashicorp.com.
 
+### `--tf-disable-downloads`
```

Comment:
Why would you disable tf downloads? Why not just provide a `--tf-download-url` to download terraform from a custom endpoint?

Reply:
Oh nvm. I see it mentioned in the PR description.

Reply:
In our case, we don't have a "custom endpoint" that we can download Terraform versions from. Instead, we sync these from S3 in a pre-workflow hook. So, in our case, it makes sense to entirely disable downloads.